src/main/java/seedu/address/model/student/Student.java

```diff
@@ -64,6 +72,19 @@ public String getRelationship() {
         return relationship;
     }
 
+    public List<Session> getListOfSessions() {
+        return this.sessions;
+    }
+
+    /**
+     * Adds a tuition session to the student.
+     * @param session to be added.
+     */
+    public void addSession(Session session) {
+        requireAllNonNull(session);
+        this.sessions.add(session);
+    }
+
     /**
      * Returns true if both students have the same name.
```

Comment:
I think the javadoc comment here should change?

Reply:
Did you mean line 80?

Reply:
Woops, I was using my iPad to review just now, and I might not have seen the code accurately. No issues!